charts/plugin-br-pix-switch/templates/ingress-apps.yaml

```
@@ -0,0 +1,63 @@
+false or no routes resolve to enabled components.
+
+No path rewriting is performed at the ingress — apps own their /<component>
+path namespace (e.g. SPI's router registers /spi/health, /spi/keys/lookup).
+*/}}
+{{- if .Values.appsIngress.enabled }}
+{{- $ctx := . }}
+{{- $cfg := .Values.appsIngress }}
+{{- $fullName := include "plugin-br-pix-switch.componentFullname" (dict "context" $ctx "component" "apps") }}
+{{- /* Collect routes whose target component is enabled. */}}
+{{- $renderedRoutes := list }}
+{{- range $route := $cfg.routes }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)